doc/bspwm.1

```diff
@@ -992,7 +992,7 @@ Manage all the unmanaged windows remaining from a previous session\&.
 .PP
 \fB\-h\fR, \fB\-\-record\-history\fR on|off
 .RS 4
-Enable or disable the recording of node focus history\&.
```

Comment:
You should edit the asciidoc file, not the `man` page directly: this change will be overwritten when the `man` page is recompiled.